examples/patch-priors/demo_epll.py

```diff
@@ -27,7 +26,8 @@
 url = get_image_url("CBSD_0010.png")
 test_img = load_url_image(url, grayscale=False).to(device)
 patch_size = 6
-model = EPLL(channels=test_img.shape[1], patch_size=patch_size, device=device)
+with torch.no_grad():
```

Comment:
why is this needed?

Reply:
When we ran the make html yesterday, this demo (and others) saturated the GPU memory. 
This does not happen with torch.nograd.

Reply:
ok, but unless I'm mistaken the one below should be sufficient?